src/data/real/ennreal.lean

```diff
@@ -373,6 +373,14 @@ lemma coe_le_iff : ↑r ≤ a ↔ (∀p:ℝ≥0, a = p → r ≤ p) := with_top.
 
 lemma lt_iff_exists_coe : a < b ↔ (∃p:ℝ≥0, a = p ∧ ↑p < b) := with_top.lt_iff_exists_coe
 
+lemma to_real_mono {a : ℝ≥0∞} {b : ℝ≥0} (h : a ≤ b) : a.to_real ≤ b :=
```

Comment:
The PR looks good to me. I have only one last remark: that lemma name feels strange. Reading `to_real_mono`, I expect a statement of the form `a.to_real \le b.to_real`. What you have here is a less straightforward statement about both ennreal and nnreal. I have trouble finding a good alternative name for this. Something like `to_real_le_coe` maybe, or the longer but more explicit `ennreal_to_real_le_nnreal_coe`? These don't sound very good either.

Reply:
Makes sense -- we are presumably in the `ennreal` namespace, though, perhaps we can drop the `ennreal` from the name.  What do you think of `ennreal.to_real_le_coe_of_le_coe`?

Reply:
Good, let's use that name!

Reply:
Done!